src/app/container/launch/launch.component.html

```diff
@@ -38,10 +38,15 @@
       <pre>{{ cli }}</pre>
     </div>
     <div *ngIf="(currentDescriptor | lowercase) === 'cwl'">
-      <div class="well well-sm" tooltip="Commands for launching tools through CWLtool: the CWL reference implementation.
-      cwltool can run a tool directly when all inputs and outputs are available on the local filesystem.">
-        <a href="https://github.com/common-workflow-language/cwltool">Run locally with cwltool</a>
+      <div class="well well-sm" tooltip="Commands for launching tools through cwl-runner when all inputs and outputs are available on the local filesystem.">
+        <a href="https://github.com/common-workflow-language/cwltool">Run locally with cwl-runner (you can use '--non-strict' when the executor is cwltool)</a>
```

Comment:
Might want to explain why someone might want to do this and if it ends up being too long, maybe it can be hidden by a drop-down, info button, etc.